library/Icinga/Cli/Command.php

```diff
@@ -45,6 +48,9 @@ abstract class Command
 
     protected $defaultActionName = 'default';
 
+    /** @var string Module loading mode. Either {@link Command::LOAD_ENABLED} or false to disable module loading */
+    protected $loadModules = self::LOAD_ENABLED;
```

Comment:
A use-case please, otherwise I'll demand this to be just `true` by default.

Reply:
We had a discussion about introducing something like `LOAD_DEPENDENCIES_ONLY` as well. So I started with the const `LOAD_ENABLED`. But I'm fine with only `true` for the moment.

Reply:
Yes, though we ditched it because we have no proper mechanism to detect dependencies. And I suspect this will change in the foreseeable future..